src/ir_Gree.cpp

```diff
@@ -117,49 +115,48 @@ IRGreeAC::IRGreeAC(const uint16_t pin, const gree_ac_remote_model_t model,
 }
 
 /// Reset the internal state to a fixed known good state.
-void IRGreeAC::stateReset(void) {
+void IRGreeAC::stateReset() {
```

Comment:
Please maintain/keep/use `(void)` instead of `()` for function parameter definitions.
That goes for all occurrences where you've changed it, not just this one.
See: https://stackoverflow.com/questions/693788/is-it-better-to-use-c-void-arguments-void-foovoid-or-not-void-foo